include/roboteam_utils/HalfLine.h

```diff
@@ -18,19 +18,19 @@ class HalfLine {
 
     public:
         /**
-         * Construct a new HalfLine object
-         * @param start The location where the HalfLine starts
-         * @param direction This vector indicates in which direction the HalfLine is infinite long
+         * Construct a new HalfLine object by using 2 different points on that Line. Make sure that the given points are different.
```

Comment:
I agree to improve the documentation, however for direction I don't have a better name (I rather don't use end as variable name, because it is even more confusing). So if you have a better name tell me.